devices/xiaomi.js

```diff
@@ -2403,6 +2527,29 @@ module.exports = [
             await endpoint.read('aqaraOpple', [0x040a], {manufacturerCode: 0x115f});
         },
     },
+    {
+        zigbeeModel: ['aqara.feeder.acn001'],
+        model: 'ZNCWWSQ01LM',
+        vendor: 'Xiaomi',
+        description: 'Aqara Pet Feeder C1',
+        fromZigbee: [fzLocal.aqara_feeder],
+        toZigbee: [tzLocal.aqara_feeder],
+        exposes: [
+            exposes.enum('feeding', ea.STATE_SET, ['START']).withDescription('Start feeding'),
+            exposes.text('feeding_source', ea.STATE).withDescription('Feeding source'),
+            exposes.text('feeding_size', ea.STATE).withDescription('Feeding size'),
+            exposes.numeric('portions_per_day', ea.STATE).withDescription('Portions per day'),
+            exposes.numeric('weight_per_day', ea.STATE).withDescription('Weight per day').withUnit('g'),
+            exposes.binary('alarm', ea.STATE, true, false).withDescription('Something wrong with feeder'),
+            exposes.text('schedule', ea.STATE_SET).withDescription('Schedule string'),
```

Comment:
what is the expected format?

Reply:
Example: `7f07000800,7f13000100`
Need to be exposed as list of time and serving size

Reply:
A list would make more sense indeed. I remember we discussed this before but it might be a good time to add it now. I see that here is already a list expose available which should also work in the frontend:

https://github.com/Koenkk/zigbee-herdsman-converters/blob/fb0b50bdff5b789149b1915d04439e11b4ed97d3/lib/exposes.js#L117

Reply:
yes, we discussed. now there is a list of numbers, but an editable list of objects is needed.